src/manifold/src/quickhull.cpp

```diff
@@ -830,14 +830,10 @@ void Manifold::Impl::Hull(VecView<vec3> vertPos) {
 
   QuickHull qh(vertPos);
   std::tie(halfedge_, vertPos_) = qh.buildMesh();
-  meshRelation_.originalID = ReserveIDs(1);
   CalculateBBox();
   SetPrecision(bBox_.Scale() * kTolerance);
-  SplitPinchedVerts();
   CalculateNormals();
   InitializeOriginal();
-  CreateFaces({});
-  SimplifyTopology();
```

Comment:
FYI @Kushal-Shah-03 I simplified the hull setup - hopefully improves the speed a bit.